src/gui/mod.rs

```diff
@@ -1845,6 +1845,7 @@ impl std::fmt::Display for CatalogSource {
                 catalog::Source::Tukui => "Tukui",
                 catalog::Source::WowI => "WowInterface",
                 catalog::Source::TownlongYak => "Townlong-Yak",
+                catalog::Source::Other => panic!("Unspported catalog source"),
```

Comment:
Should be fine to `panic!` in the `std::fmt::Display` since we don't want to render this to the GUI.

Reply:
Yup! The only way it'll expose itself in the GUI and make allow any catalog items for it to display is if you add it to the CatalogSource::all method. Since we left it out, these should be unreachable and it'd be impossible to show those items in the GUI.